src/index.html

```diff
@@ -7,8 +7,72 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500&display=swap" rel="stylesheet">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="conteiner" id="home">
+      <li class="nav__item">
+        <a  href="#home">
+          <img
+          class="img"
+          src="images/logo.png"
+          alt="Logo MOYO"
+        >
+        </a>
+      </li>
+      <nav class="navigation">
+        <navigaion__list>
+          <li>
+            <a class="nav__link is-active"
+               href="."
+          >Apple</a>
+          </li>
+        </navigaion__list>
+        <navigaion__list>
+         
+          <a class="nav__link" href=".">SAMSUNG</a>
+      
+        </navigaion__list>
+        <navigaion__list>
+        
+          <a class="nav__link" href=".">SMARTPHONES</a>
+     
+        </navigaion__list>
+        <navigaion__list>
+         
+          <a class="nav__link" 
+           href="."
+           data-qa="hover"
+            >
+           LAPTOPS & COMPUTERS
+          </a>
+      
+        </navigaion__list>
+
+        <navigaion__list>
+        
+          <a class="nav__link" href=".">GADGETS</a>
+         
```

Comment:
you need to cover all these links in `li` tag and `li` tags put in `ul` tag with class name `navigation__list`